header-verifier/tests/header_verifier_blackbox_setup.rs

```diff
@@ -1,19 +1,16 @@
+use common_blackbox_setup::{AccountSetup, BaseSetup, HEADER_VERIFIER_ADDRESS};
 use multiversx_sc::types::ManagedBuffer;
 use multiversx_sc::{
     api::ManagedTypeApi,
-    types::{BigUint, MultiValueEncoded, TestAddress, TestSCAddress},
+    types::{MultiValueEncoded, TestAddress},
 };
 use multiversx_sc_scenario::scenario_model::Log;
 use multiversx_sc_scenario::{
-    api::StaticApi, imports::MxscPath, multiversx_chain_vm::crypto_functions::sha256,
-    ScenarioTxRun, ScenarioWorld,
+    api::StaticApi, multiversx_chain_vm::crypto_functions::sha256, ScenarioTxRun,
 };
 use multiversx_sc_scenario::{ReturnsHandledOrError, ReturnsLogs};
 use proxies::header_verifier_proxy::HeaderverifierProxy;
 
-const HEADER_VERIFIER_CODE_PATH: MxscPath = MxscPath::new("ouput/header-verifier.mxsc-json");
-pub const HEADER_VERIFIER_ADDRESS: TestSCAddress = TestSCAddress::new("header-verifier");
-
 // NOTE: This is a mock path
 pub const ENSHRINE_ADDRESS: TestAddress = TestAddress::new("enshrine");
```

Comment:
I guess you could use the constants from common blackbox setup for all of these.